source/js/script.js

```diff
@@ -134,18 +134,28 @@
     return el.childNodes[0];
   }
 
-  // Grab first character for dropcaps
```

Comment:
Removing this will affect all other themes that use this old method. I'm wondering if we should leave this in longer to enabled the ModX theme to make this change?

Reply:
@designerbrent 
I'll review to see if leaving this in causes any conflicts. If not, then that makes sense - no need to remove it if it would be a breaking change for some cases.

Reply:
Any word on this, @davideleuterius?